boot/freeldr/freeldr/include/arch/pc/pcbios.h

```diff
@@ -3,24 +3,59 @@
 
 #ifndef __ASM__
 
+// 2 following BIOS_MEMORY_* updated to ACPI 6.2-A.
```

Comment:
Change logs don't belong in source code.

Reply:
Removed.